mozilla/foundation.mozilla.org
# PNI products inheriting share image

Closes #7165 
Related PRs/issues #

### **Link to sample test pages and Metadata preview**:


**Test PNI homepage, with custom meta description and image:**
https://foundation-s-pni-share--4dl0xc.mofostaging.net/en/privacynotincluded/
<img width="691" alt="Screen Shot 2021-10-20 at 6 38 19 PM" src="https://user-images.githubusercontent.com/18314510/138196383-8ac9e642-3529-4842-b0c2-6e97ec66b66f.png">

**Test PNI Product, with custom meta description and image:**
https://foundation-s-pni-share--4dl0xc.mofostaging.net/en/privacynotincluded/minute-dinner-also/
<img width="691" alt="image" src="https://user-images.githubusercontent.com/18314510/138196424-6ff6fbf7-cb1e-4768-bc93-e2628268cccf.png">

**Test PNI product with no custom metadata set, should inherit from PNI homepage:**
https://foundation-s-pni-share--4dl0xc.mofostaging.net/en/privacynotincluded/gas-word-mr-front-involve-street-hold/
<img width="691" alt="image" src="https://user-images.githubusercontent.com/18314510/138196476-2be647ef-3c7d-4b2c-a524-2e08dc0663b4.png">

**Steps to test:**

1. Copy and paste the links above in a messaging app of your choice, and the metadata should match the images next to each URL.
2. Please note that the third link, has no metadata set, which means it is inheriting the metadata set from the PNI home page. If we were to set either description or image, it would appear here.
3. If everything is working as expected, testing is complete! 

## Checklist

**Changes in Models:**
